.github/workflows/actions/manage-doppler-secret/manage.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
```

Comment:
Clean shellcheck! Nice!

Reply:
Yep! Thanks to your tip for installing that plugin.